src/main/scala/io/iohk/ethereum/consensus/ethash/validators/BlockHeaderValidatorImpl.scala

```diff
@@ -0,0 +1,82 @@
+package io.iohk.ethereum.consensus.ethash
+package validators
+
+import akka.util.ByteString
+import io.iohk.ethereum.crypto
+import io.iohk.ethereum.domain.BlockHeader
+import io.iohk.ethereum.utils.BlockchainConfig
+import io.iohk.ethereum.validators.BlockHeaderError.HeaderPoWError
+import io.iohk.ethereum.validators.BlockHeaderValidatorImpl.PowCacheData
+import io.iohk.ethereum.validators.{BlockHeaderError, BlockHeaderValid, BlockHeaderValidator}
+
+// NOTE Copied parts from [[io.iohk.ethereum.validators.BlockHeaderValidatorImpl]]
+class BlockHeaderValidatorImpl(blockchainConfig: BlockchainConfig) extends BlockHeaderValidator {
```

Comment:
The validators will be refactored in the next iteration, right? Will we still need 2 complementary implementations of `BlockHeaderValidator`? (I understand we can have even more, each specific to a given protocol, I'm not sure though about having 2 different ones for a single protocol)